openshift/4.0/templates/app/deploy.yaml

```diff
@@ -296,6 +299,8 @@ objects:
                 - name: "${SOLUTION_NAME}-${APP_NAME}-envvars"
                   mountPath: "${FILE_CONFIG_MOUNT_PATH}environment.json"
                   subPath: "environment.json"
+                - name: file-storage
+                  mountPath: /tmp/app/dist/videos
```

Comment:
just curious, would this mean we would need to upload all the videos to this folder in each of the dev, test, and prod environments and if so how would we do it?

Reply:
We just upload the videos with `oc rsync` util.  I've uploaded to DEV already, so you can already view the videos manually.  Once the PR is merged we can test the links.